src/components/CustomerQuota/ItemsSelection/ItemIdentifiersCard.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FunctionComponent, useEffect } from "react";
+import { PolicyIdentifierInput } from "../../../types";
+import { CartHook, CartItem } from "../../../hooks/useCart/useCart";
+import { ItemIdentifier } from "./ItemIdentifier";
+import { View, StyleSheet } from "react-native";
+import { size } from "../../../common/styles";
+
+const styles = StyleSheet.create({
+  content: {
+    position: "relative",
+    padding: size(2),
+    width: 512,
+    maxWidth: "100%"
+  }
+});
+
+export const ItemIdentifiersCard: FunctionComponent<{
+  cartItem: CartItem;
+  identifierInputs: PolicyIdentifierInput[];
+  setIdentifierInputs: (input: PolicyIdentifierInput[]) => void;
+  updateCart: CartHook["updateCart"];
+}> = ({ cartItem, identifierInputs, setIdentifierInputs, updateCart }) => {
+  const updateIdentifierValue = (index: number, value: string): void => {
+    setIdentifierInputs([
+      ...identifierInputs.slice(0, index),
+      { ...identifierInputs[index], value },
+      ...identifierInputs.slice(index + 1)
+    ]);
+  };
+
+  useEffect(() => {
+    updateCart(cartItem.category, cartItem.quantity, identifierInputs);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [identifierInputs]);
```

Comment:
Including all deps was causing state to change too fast, so checking out became unpredictable:
* Sometimes the identifiers were not recorded when checking out
* Sometimes can't check out even after filling in the identifiers

Not sure if there is a better way to solve this!! What do you think?

Reply:
oh, serious? hmm, so adding the lint fixes it ah? :O

Reply:
Lint keep saying need to add `cartItem.X` dependencies in too, but doesn't make sense cuz they won't change 😆  Adding lint avoids the warning so we can only add `identifierInputs` to deps array

Reply:
ohh, i see i see! hmm yeah, maybe we can think about it, to see if there's a better solution. I need to think through also HAHA